super_rust_boy_c_binding/src/lib.rs

```diff
@@ -64,7 +64,7 @@ pub extern fn rustBoyCreate(cartridge_path: *const c_char, save_file_path: *cons
         }
     };
 
-    let instance = RustBoy::new(cart_path, save_path, UserPalette::Default, false);
+    let instance = RustBoy::new(cart_path, save_path, UserPalette::Default, true);
```

Comment:
Would be cool to not mute on macOS

Reply:
This change was already merged in the other PR.

Reply:
Would it be possible to take the mute parameter from the outside so on macos builds we can not mute?

Reply:
Yes! I'll look into it

Reply:
@coopersimon Exposed mute now